src/components/MerchantsPage/gam/styles.module.scss

```diff
@@ -19,7 +19,7 @@
   grid-template-rows: 300px 215px;
 }
 
-.main {
+.header_text {
```

Comment:
nit: I think we've been using Pascal Case for our classes, not snake--but no need to change unless you're doing something else too

Reply:
Noted. I can definitely take a look over and fix these.